src/next-transpile-modules.js

```diff
@@ -36,31 +40,47 @@ const regexEqual = (x, y) => {
   );
 };
 
+/**
+ * Return the root path (package.json directory) of a given module
+ * @param {string} module
+ */
+const getPackageRootDirectory = (module) => {
+  let packageDirectory;
+  let packageRootDirectory;
+
+  try {
+    // Get the module path
+    packageDirectory = resolve(CWD, module);
```

Comment:
This will only look into the current working directory correct? What about when using yarn workspaces and some packages are hoisted outside the cwd? Seems to error out

Reply:
Answer line 63 ;)